src/components/Countdown.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC, useState } from 'react';
+import { Text, StyleSheet, View } from 'react-native';
+
+import { DateType } from 'app/consts';
+import { getTimeDiff, getTimeDuration } from 'app/helpers/date';
+import { useInterval } from 'app/helpers/useInterval';
+import { palette, fonts } from 'app/styles';
+
+const i18n = require('../../loc');
+
+interface CellsProps {
+  text: string;
+  title: string;
+}
+
+const Cells: FC<CellsProps> = ({ text, title }) => (
```

Comment:
Why it's named "Cells" in the plural form? Wouldn't be better to name it just "Cell"?

Reply:
It should be plural because a single cell is a single letter and here we are rendering multiple letters